packages/app/components/Queue/Student/StudentQuestionRephraseModal.tsx

```diff
@@ -0,0 +1,35 @@
+import { RephraseQuestionPayload } from "@koh/common";
+import { Button } from "antd";
+import Modal from "antd/lib/modal/Modal";
+import React, { ReactElement } from "react";
+
+type StudentQuestionRephraseModalProps = {
+  courseId: number;
+  payload: RephraseQuestionPayload;
+  handleClose: (courseId: number, queueId: number) => void;
+};
+export default function StudentQuestionRephraseModal({
+  courseId,
+  payload,
+  handleClose,
+}: StudentQuestionRephraseModalProps): ReactElement {
+  return (
+    <Modal
+      visible={true}
+      footer={[
+        <Button
+          type={"primary"}
+          key={"continue"}
+          onClick={() => handleClose(courseId, payload.queueId)} // TODO: replace courseId with courseId from payload
```

Comment:
Same thing here, course should match the course that the alert originated from, this courseId is coming from the current course in the navbar (not what we want)